frontend/src/components/LoginForm.tsx

```diff
@@ -60,9 +62,29 @@ export default function LoginForm({
   const history = useHistory()
 
   const { from } = location.state || { from: { pathname: "/" } }
+  const [isOpen, setOpen] = React.useState<boolean>(false)
+  const [type] = useState<NotificationType["type"]>("error")
+  const [message] = useState("Email and/or password entered incorrectly")
```

Comment:
These states never change and can therefore just be constant.
```suggestion
  const type: NotificationType["type"] = "error"
  const message = "Email and/or password entered incorrectly"
```

Reply:
this isn't a file I touched, I'm not sure I should make this change in this PR

Reply:
Weird. I don't know then. The behavior should be the same either way.